packages/babel-core/src/transformation/plugin.js

```diff
@@ -1,15 +1,12 @@
 import OptionManager from "./file/options/option-manager";
 import * as messages from "babel-messages";
-import Store from "../store";
 import traverse from "babel-traverse";
 import clone from "lodash/clone";
 
 const GLOBAL_VISITOR_PROPS = ["enter", "exit"];
 
-export default class Plugin extends Store {
```

Comment:
Ah ok I couldn't tell so I just assumed it was Plugin (and of course the tests passes so it's good, just was confused)